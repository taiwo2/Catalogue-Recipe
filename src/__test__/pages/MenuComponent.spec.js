import React from 'react';
import renderer from 'react-test-renderer';
import MenuComponent from '../../components/MenuComponent';

test('should correctly render Single page', () => {
  const component = renderer.create(<MenuComponent />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
