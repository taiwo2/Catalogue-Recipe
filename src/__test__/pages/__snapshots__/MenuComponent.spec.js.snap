// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Single page 1`] = `
<button
  className="btn-icon"
  onClick={[Function]}
  type="button"
>
  <img
    alt="cart"
    className="menu-icon"
    src="menu.png"
  />
</button>
`;
